### DISCUSSION TITLE
OVPN profiles do not work on new OpenVPN clients.

### DISCUSSION THREAD

**Discussion by nanedi** [angristan/openvpn-install#1408] [Help]
OVPN profiles do not work on new OpenVPN clients.

There is no traffic where the VPN connection is successful.

Why?

Thank you.

**Comment by iRakic** [comment]
I have the same issue. Basically, it works on MacOS, but on Windows clients, it doesn't work.
@nanedi, have you found a solution for this issue?

**Comment by nanedi** [comment]
> I have the same issue. Basically, it works on MacOS, but on Windows clients, it doesn't work. @nanedi, have you found a solution for this issue?

Reinstall newer OS; 
Reinstall OpenVPN;
Select a compression method.
